### PR TITLE
cargo-flamegraph: init at 0.1.13

### PR DESCRIPTION
**Pull request by killercup**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by killercup** [comment]
I tested this on NixOS before rebasing.

**Comment by xrelkd** [comment]
@GrahamcOfBorg build cargo-flamegraph

File: pkgs/development/tools/cargo-flamegraph/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-flamegraph";
+  version = "0.1.13";
+
+  src = fetchFromGitHub {
+    owner = "ferrous-systems";
+    repo = "flamegraph";
```
Comment (worldofpeace) [inline pkgs/development/tools/cargo-flamegraph/default.nix]:
```suggestion
    repo = pname;
```

**Comment by worldofpeace** [comment]
Looks good, going to run the builder again for darwin
@GrahamcOfBorg build cargo-flamegraph

**Review by worldofpeace — APPROVED** [review]
LGTM with the above comment

File: pkgs/development/tools/cargo-flamegraph/default.nix (hunk 1 above)
Comment (killercup) [reply pkgs/development/tools/cargo-flamegraph/default.nix]: The names differ -- `pname` is "cargo-flamegraph".

Comment (worldofpeace) [reply pkgs/development/tools/cargo-flamegraph/default.nix]: Right, thanks for noticing.

**Pull request merged**